charts/teuto-portal-k8s-worker/templates/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" $ | nindent 6 }}
+  template:
+    metadata:
+      labels: {{- include "common.labels.standard" $ | nindent 8 }}
+    spec:
+      {{- include "common.images.renderPullSecrets" ( dict "images" (list .Values.worker.image) "context" $) | indent 6 }}
+      securityContext: {{- toYaml .Values.global.securityContext | nindent 8 }}
+      automountServiceAccountToken: false
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext: {{- toYaml .Values.global.podSecurityContext | nindent 12 }}
+          image: {{ template "portalworker.image". }}
+          {{- if .Values.worker.image.digest }}
+          imagePullPolicy: IfNotPresent
+          {{- else }}
+          imagePullPolicy: Always
+          {{- end }}
+          env:
+            - name: JDBC_URL
+              value: {{ include "portalworker.jdbc" . | quote }}
+            - name: DB_PORT
+              value: {{ .Values.worker.database.port | quote }}
+            - name: KUBECONFIG
+              value: "/kubeconfig"
+            {{- with (include "portalworker.dbcredentials" $) }}
+            - name: DB_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ . }}
+                  key: username
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ . }}
+                  key: password
+            {{- end }}
+            {{- range $k, $v := .Values.worker.config }}
+            {{- if or (typeIs "bool" $v) ($v) }}
+            - name: {{ regexReplaceAll `_(\d+)` ($k | snakecase | upper) "$1" }}
+              value: {{ $v | quote }}
```

Comment:
This doesn't work for the `flavourPrefixFilter`, it results in `"[teuto teuto-net]"` but it should be `"teuto,teuto-net"`.

I guess you should create an `if` with a `typeis "slice"` (I think) and join them.

Maybe just adding `| join ","` in the pipe before the quote could work 🤔

Reply:
the `| join ","` did the trick